lib/raven/configuration.rb

```diff
@@ -198,7 +198,7 @@ def initialize
       self.project_root = detect_project_root
       self.rails_activesupport_breadcrumbs = false
       self.rails_report_rescued_exceptions = true
-      self.release = detect_release
+      self.release = Utils::Release.new(logger).detect_release
```

Comment:
Perhaps the 30 AbcSize does not allow getting rid of the `detect_release` method.